src/components/ManageUsers/UserSearch.jsx

```diff
@@ -0,0 +1,23 @@
+const UserSearch = () => {
+  return (
+    <div className="usersearch__container">
+      <div>
+        <label className="usersearch__label">Enter user email:</label>
```

Comment:
label must have a `for` attribute corresponding with input id, so in your case it will be:
```suggestion
        <label className="usersearch__label" for="search">Enter user email:</label>
     
```
and I guess the div around the label is redundant